payment-channels-htlcs/README.md

```diff
@@ -15,30 +15,58 @@
+### History of the Lightning Network
+
 1. Why was Segwit an important upgrade to the base layer for Lightning?
-1. How are accidental breaches handled on Lightning (since at the moment this is mostly what is seen in the the wild)?
+1. Why is restoring a non-up-to-date backup unsafe with Lightning? How recent does the backup need to be?
+1. How can we prepare and anticipate for miners fees volatility when creating new payment channels? (What is the mechanic?)
+1. What is different about transaction revocation in Eltoo?
 
-### HTLCs
+### Lightning ≈ Bitcoin
 
-1. Is it correct to say that under "normal" working conditions, if all payments are successful, HTLC outputs are merged back to the main outputs? Meaning the HTLC portion of the commitment transaction rarely gets recorded on-chain?
-1. Does the dust limit of the base layer affect the security of HTLCs routed on Lightning?
+1. Given the fact that Lightning is punishment-based and requires nodes to be online, could an attacker trick you into publishing old states and lose all your funds?
+1. How are accidental breaches handled on Lightning (since at the moment this is mostly what is seen in the the wild)?
 
-### Breach Remedy
+### Payment Channels video | Understanding Payment Channels article
 
+1. Why is a unidirectional channel easy to implement?
+1. What is the benefit of the asymmetric commitment transactions?
 1. When two parties exchange the previous commitment's secrets (to invalidate previous state), how do you make sure that the exchange happen atomically? (i.e., that both receive the other's secret, or none at all)
 1. How do you exchange previous commitment data if the parties aren't online on the same time?
+1. Would it be possible for channels to have a custom penalty agreement that does not take all the funds on a breach?
 
-### CHECKSIGFROMSTACK / OP_PUSHTXDATA
+### Revocable Transactions
 
-1. How can signature checks of arbitrary data be used to avoid storing signatures for each channel state?
+1. Why can't a transaction have an expiration date or maximum blockheight?
+1. Why is revoking a transaction in bitcoin tricky? Can a user just take back a transaction?
+1. How is the commitment secret used to create the breach remedy transaction?
 
-### Timelocks
+### Bitcoin Timelocks Explained
 
 1. Why do CSV and CLTV rely on nLocktime or nSequence being set in the transaction spending the outputs when the script could check if the requirements are being met by itself?
 1. Can both CSV and CLTV be used in the same output and are there any known use cases for it?
 1. Is it fair to say UTXO-level timelocks are superior to transaction-level timelocks since they are baked directly into the protocol and allow more flexiblity and control? Are transaction-level timelocks necessary (Assuming there's a way to implement UTXO-level timelocks without relying on nLockTime/nSequence)?
+1. In what scenarios is OP_CLTV used in Lightning and in what scenarios is OP_CSV used?
```

Comment:
Not a strong opinion, but maybe put the OPs in code fences like `OP_CLTV`/`OP_CSV`